### PR TITLE
permissions: Add panel to settings screen

### LINKED ISSUE #585: Permission settings are missed on the settings page
Screenshot 2020-11-06 at 14 59 27

### PR DESCRIPTION
**Pull request by etharner**
Closes #585.
Please correct wording if needed.

### CODE REVIEW

File: src/popup/router/pages/Settings.vue [hunk 1]
```diff
@@ -16,6 +16,11 @@
         :title="$t('pages.appVUE.networks')"
         :info="$t('pages.settings.tabNetworksSmall')"
       />
+      <PanelItem
+        to="/permissions-settings"
```
Comment (davidyuk) [inline src/popup/router/pages/Settings.vue:20]: I would refer to the route by name, but it is ok to merge it as it is.

File: src/popup/router/components/PanelItem.vue [hunk 2]
```diff
@@ -74,6 +74,7 @@ export default {
   .panel-item-info {
     color: #ccc;
     font-weight: normal;
+    line-height: normal;
```
Comment (davidyuk) [inline src/popup/router/components/PanelItem.vue:77]: Don't see a reason to add this.

Comment (davidyuk) [reply src/popup/router/components/PanelItem.vue:77]: | With | Without |
|---|---|
| Screenshot 2020-11-17 at 14 33 27 | Screenshot 2020-11-17 at 14 33 23 |

To fir to some phone screen?🤔

Comment (etharner) [reply src/popup/router/components/PanelItem.vue:77]: There are different line-height on button (Remove account) and a link (Permissions settings and upper panels). Browser set it by default for buttons and it rewrites our line height defined for html in https://github.com/aeternity/superhero-wallet/blob/develop/src/common/base.scss#L21, but for link it's not, so line-height became 1.5

Comment (mradkov) [reply src/popup/router/components/PanelItem.vue:77]: We anyway will have a different design for this page very soon, so I don't think we have to spend time refactoring it now.

Comment (etharner) [reply src/popup/router/components/PanelItem.vue:77]: @mradkov So, remove this line?

**Review by davidyuk — APPROVED** [review]

**Pull request merged**